### PR TITLE
files added

### PR DESCRIPTION
**Pull request by Raniya-thayyil**

### CODE REVIEW

File: flipkart/src/components/desktopview/Advertisement.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./AcMobiles.css";
+
+function Advertisement(props) {
+  const { adsList } = props;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/desktopview/Advertisement.js:5]: remove the space. Do not pad JSX curly braces with spaces

File: flipkart/src/components/desktopview/CarouselDesktop.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    height: 280px;
+}
+
+.carousel-desk {
+    display: flex;
+    overflow: hidden;
+    margin-top: 12px;
+    padding: 8px;
+    position: relative;
+    margin-inline: 8px;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/desktopview/CarouselDesktop.css:12]: place margins together.

File: flipkart/src/components/desktopview/CarouselDesktop.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react";
+
+import leftAngle from "../assets/leftAngle.svg";
+import "./CarouselDesktop.css";
+
+function CarouselDesktop(props) {
+  const { carouselList } = props;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/desktopview/CarouselDesktop.js:7]: remove space between braces

File: flipkart/src/components/desktopview/CarouselDesktop.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+          >
+            <img src={item} alt="carousel images" />
+          </div>
+        ))}
+        <div className="arrow-right">
+          <img src={leftAngle} onClick={handleClickRight} />
+        </div>
+      </div>
+      <div className="hdfc">
+        <img src="https://rukminim1.flixcart.com/fk-p-flap/2000/2000/image/8d8f584b6e0ddb5f.jpg?q=50" />
```
Comment (Anjugeorg-e) [inline flipkart/src/components/desktopview/CarouselDesktop.js:58]: add alternate text with img tags

File: flipkart/src/components/desktopview/FeaturedBrand.css [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    padding-inline: 20px;
+    padding-bottom: 18px;
+}
+
+.features-main h2 {
+    text-align: left;
+    font-family: font1;
+    font-size: 20px;
+    color: #212121;
+    padding-top: 18px;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/desktopview/FeaturedBrand.css:26]: try to add shorthand properties

File: flipkart/src/components/desktopview/FeaturedBrand.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
+import React, { useRef } from "react";
+
+import leftAngle from "../assets/leftAngle.svg";
+
+import "./FeaturedBrand.css";
+
+function FeaturedBrand(props) {
+  const { featuresList } = props;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/desktopview/FeaturedBrand.js:8]: avoid this space  between curly braces

File: flipkart/src/components/desktopview/TopDeals.css [hunk 7]
```diff
@@ -0,0 +1,64 @@
+.top-deal-textlink h2 {
+    color: #000000;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/desktopview/TopDeals.css:2]: try to give 3 digit  character for color

File: flipkart/src/components/desktopview/TopDeals.css [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+    font-size: 30px;
+    font-family: font1;
+    font-weight: 100;
+
+}
+
+.top-deals-main {
+    margin-top: 12px;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/desktopview/TopDeals.css:10]: arrange margins together

File: flipkart/src/components/desktopview/TopDeals.js [hunk 9]
```diff
@@ -0,0 +1,60 @@
+import React, { useRef } from "react";
+
+import leftAngle from "../assets/leftAngle.svg";
+import "./TopDeals.css";
+
+function TopDeals(props) {
+  const { dealsList } = props;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/desktopview/TopDeals.js:7]: remove space betweeen curly braces

File: flipkart/src/components/headers/Header.js [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+import './Header.css'
+
+function Header() {
+  return (
+    <>
+      <div className="header-main">
+        <div className="header-top">
+            <div className="ham-logo">
+                <img src={ham}/>
```
Comment (Anjugeorg-e) [inline flipkart/src/components/headers/Header.js:17]: add alternate texts

File: flipkart/src/components/mobileview/Carousel.js [hunk 11]
```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+
+import "./Carousel.css";
+
+function Carousel(props) {
+  let [index, setIndex] = useState(0);
+
+  const { carouselList } = props;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/Carousel.js:8]: space ?

File: flipkart/src/components/mobileview/ElectronicList.css [hunk 12]
```diff
@@ -0,0 +1,12 @@
+.electronic-list {
+    display: grid;
+    grid-template-columns:repeat(3, 1fr);
+
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/ElectronicList.css:4]: remove unwanted spaces

File: flipkart/src/components/mobileview/InterestingFind.css [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+    height: 427px;
+    justify-content: center;
+}
+
+.list-items {
+    background-color: #fff;
+    padding: 16px 16px 12px;
+    border: 1px solid rgb(248, 243, 243);
+    font-family: font4;
+
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/InterestingFind.css:35]: why space added here ?

File: flipkart/src/components/mobileview/InterestingFind.js [hunk 14]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./InterestingFind.css";
+
+function InterestingFind(props) {
+  const { findsList } = props;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/InterestingFind.js:5]: space?

File: flipkart/src/components/mobileview/TwoImages.js [hunk 15]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import "./TwoImages.css";
+
+function TwoImages(props) {
+  const { twoImagesList } = props;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/TwoImages.js:6]: remove space

File: flipkart/src/components/mobileview/Whishlist.js [hunk 16]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import "./Whishlist.css";
+
+function Whishlist(props) {
+  const { alerts } = props;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/Whishlist.js:6]: you have addede the same mistakes multiple times. so please careful about this..

File: flipkart/src/components/mobileview/ThreeImages.js [hunk 17]
```diff
@@ -0,0 +1,21 @@
...
+function ThreeImages(props) {
+  const {
+    threeImagesList
+  } = props
+
+  return (
+    <>
+      <div className="img-list">
+        {threeImagesList.map((item, idx) => {
+          return <img src={item.image} />;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/ThreeImages.js:14]: add alt text

File: flipkart/src/components/mobileview/Sponsored.js [hunk 18]
```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from "react";
+
+import "./Carousel.css";
+
+function Sponsored(props) {
+  let [index, setIndex] = useState(0);
+  const { sponsoredList } = props;
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/Sponsored.js:7]: space?

File: flipkart/src/components/mobileview/FirstNav.css [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+::-webkit-scrollbar {
+    display: none;
+}
+
+@media screen and (min-width:375px) {
+    .nav-list ul li img {
+        width: 68px;
+    }    
+}
+@media screen and (min-width:425px) {
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/FirstNav.css:42]: where is teh blank space?

File: flipkart/src/components/mobileview/FirstNav.css [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+    .nav-list ul li img {
+        width: 78px;
+    }    
+}
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/FirstNav.css:46]: here also no space added ?

File: flipkart/src/components/mobileview/FirstNav.css [hunk 21]
```diff
@@ -0,0 +1,51 @@
...
+
+@font-face {
+    font-family: font2;
+    src: url('../fonts/font2.woff2')
+}
+
+@font-face {
+    font-family: font3;
+    src: url('../fonts/font3.woff2')
+}
```
Comment (Anjugeorg-e) [inline flipkart/src/components/mobileview/FirstNav.css:20]: blank line is not added !!

File: flipkart/src/components/mobileview/Carousel.js (hunk 11 above)
Comment (Raniya-thayyil) [reply flipkart/src/components/mobileview/Carousel.js:8]: sorry

File: flipkart/src/components/mobileview/ElectronicList.css (hunk 12 above)
Comment (Raniya-thayyil) [reply flipkart/src/components/mobileview/ElectronicList.css:4]: ok

File: flipkart/src/components/mobileview/InterestingFind.css (hunk 13 above)
Comment (Raniya-thayyil) [reply flipkart/src/components/mobileview/InterestingFind.css:35]: sorry

File: flipkart/src/components/mobileview/Whishlist.js (hunk 16 above)
Comment (Raniya-thayyil) [reply flipkart/src/components/mobileview/Whishlist.js:6]: sorry boss

File: flipkart/src/components/mobileview/FirstNav.css (hunk 19 above)
Comment (Raniya-thayyil) [reply flipkart/src/components/mobileview/FirstNav.css:42]: here

**Pull request merged**